ci/jenkins/jobs/projects.yaml

```diff
@@ -386,6 +557,92 @@
           started_status: null
           wrappers: []
           publishers: []
+      - '{name}-{test_name}-for-pull-request':
+          test_name: whole-conformance-coverage-pending-label
```

Comment:
I think we don't need a pending-label job for whole-conformance, just like the original one. We usually don't trigger it.